Plugins/NativeCamera/Source/Android/NativeCameraImpl.h

```diff
@@ -29,8 +29,7 @@ namespace Babylon::Plugins
 
         bool m_overrideCameraTexture;
 
-        uint32_t m_width{};
-        uint32_t m_height{};
+        Camera::CameraDimensions m_cameraDimensions{};
```

Comment:
So basically we are using this to store the max width/height, then overriding it with the actual width/height after the open call. I wonder if it would make core sense to store the original camera constraints, and just return the CameraDimensions. With the WebAPI, I think you can dynamically change some constraints and restart the video or something, in which case it would be better to just store the constraints. I think we could change this later though too.

Reply:
I think we would want to have a constraints object that is stored separately (or as part of the CameraDimensions), but as it is right now we don't have any way to update the constraints during the session.